frontend/src/components/UnitAvatar.tsx

```diff
@@ -59,6 +65,8 @@ const UnitAvatar: React.FunctionComponent<Props> = ({ unit }: Props) => {
               ${gradientPhases}
             ), 
             conic-gradient(red 0turn, red ${hpPercentage}turn, gray ${hpPercentage}turn, gray 1turn)`,
+        width: `${!!size ? size : 90}px`,
```

Comment:
defaultSize is 80px but here it's 90px? Anyway, it's expanding the gameboard cell when placing a unit, which shouldn't happen

Reply:
It's still expanding the gameboard - by little, but it does.

Reply:
Hmm, seems to differ across browsers. In theory there's 72px - 10px (padding) = 62px of space for the avatar. The current height is set to 60px. I further reduced it to 58px and it works fine on chrome now.